docs/guides/mfa/20_webauthn.mdx

```diff
@@ -0,0 +1,175 @@
+---
+id: webauthn
+title: WebAuthn / FIDO2
+sidebar_title: WebAuthn / FIDO2 / YubiKey
```

Comment:
```suggestion

```

@aeneasr I think it's better to identify the technology here by the names of the standards behind it rather than a specific product. After all, there are more products like YubiKey and people might think we're being paid for putting the spotlight on YubiKey.

Reply:
Maybe also in hyphens? YubiKey's are quite well known now and FIDO2 might not say too much to most (newbie) users. WDYT?

```suggestion
sidebar_title: WebAuthn / FIDO2 (YubiKey)
```

Reply:
Sure, if you're not afraid of accusations of being sponsored then I'm cool with that 👌🏻